samples/tutorial/provider/src/provider_impl.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use digital_twin_model::sdv_v1 as sdv;
+use log::{debug, info};
+use samples_protobuf_data_access::tutorial_grpc::v1::digital_twin_provider_tutorial_server::DigitalTwinProviderTutorial;
+use samples_protobuf_data_access::tutorial_grpc::v1::{
+    GetRequest, GetResponse, InvokeRequest, InvokeResponse,
+};
+use tonic::{Request, Response, Status};
+
+#[derive(Debug, Default)]
+pub struct ProviderImpl {}
+
+#[tonic::async_trait]
+impl DigitalTwinProviderTutorial for ProviderImpl {
+    /// Get implementation.
+    ///
+    /// # Arguments
+    /// * `request` - Get request.
+    async fn get(&self, request: Request<GetRequest>) -> Result<Response<GetResponse>, Status> {
+        let request_inner = request.into_inner();
+        let entity_id: String = request_inner.entity_id.clone();
+
+        let mut get_response: GetResponse = GetResponse { property_value: String::from("NULL") };
```

Comment:
github comments aren't working properly for me right now so I can't do a multiline suggestion, but consider a match statement for lines 26-32:

```rust
let value = match entity_id.as_str() {
    sdv::hvac::ambient_air_temperature::ID => "42",
    sdv::hvac::is_air_conditioning_active::ID => "true",
    _ => "NULL",
};

let get_response = GetResponse { property_value: String::from(value) };
```